app/src/main/java/com/jerboa/ui/components/login/Login.kt

```diff
@@ -201,7 +211,7 @@ fun LoginForm(
             modifier = Modifier.padding(top = 10.dp),
         ) {
             if (loading) {
-                CircularProgressIndicator()
+                CircularProgressIndicator(modifier = Modifier.size(24.dp))
```

Comment:
Would probably be better to have this size come from the material text size in use.